src/main/java/duke/ui/MainWindow.java

```diff
@@ -0,0 +1,63 @@
+package duke.ui;
```

Comment:
It appears JavaDoc comments for the following class has yet to be added. Do remember to add them in when fine-tuning your file! 👍🏻